packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -153,7 +156,9 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
```

Comment:
How come the functions up here are not teed up to the emitter?

Reply:
Oh, this is a prototype PR that sets up all the plubming and a couple of examples that demonstrate the mechanics end-to-end. I like to do this to make it easier for me to develop and for you to review -- less repetitive changes to look at.

Once this PR lands, I'll upload a fairly mechanical PR to go add the same behaviour to all the remaining methods.